src/backend/EdFi.DataManagementService.Backend.Postgresql/Operation/SqlAction.cs

```diff
@@ -648,6 +648,132 @@ TraceId traceId
         return result;
     }
 
+    /// <summary>
+    /// Update the edfidoc of related resources when a child identity is updated
```

Comment:
I think this needs an example in the method doc of what a specific generated SQL statement would look like for a specific resource. Something like "For example, for a Session resource with identity fields abc, zyx, and pdq, the generated SQL would look like..." Then someone new to this could understand how the code should work without first reverse-engineering the behavior of `BuildUpdateStatement()`.